utils/create-keying-sheet-tfm-facilities/mock-data/deal.ts

```diff
@@ -0,0 +1,262 @@
+import { Deal } from '@ukef/dtfs2-common';
+import { ObjectId } from 'mongodb';
+
+/**
+ * Creates an active deal that can be used for utilisation reporting testing.
+ * It can be replaced by creating a deal manually and then pulling from the mongo db collection.
+ * This way we ensure we are inserting a valid deal.
+ * @param ukefDealId - ukef deal id to use
+ * @param dealId - id of deal
+ * @param portalUserId - id of a portal user
+ * @returns a deal
+ */
+export const aDeal = (ukefDealId: string, dealId: ObjectId, portalUserId: ObjectId): Deal => ({
+  _id: dealId,
+  dealType: 'GEF',
+  version: 1,
+  maker: {
+    username: 'maker1@ukexportfinance.gov.uk',
```

Comment:
could any of these come out of other files.  For example 

`const USERS = require('../portal-users');

const A_MAKER = USERS.BANK1_MAKER1;`

Reply:
I intentionally didn't do that because it would be a lot of effort and then if we ever wanted to replace with a fresh one would have to do the whole replacement job again given to ensure this is valid it is pulled from the database on dev.